### PR TITLE
fix long username issue in public pages

### LINKED ISSUE #5770: Again, submenu on public page gets broken
This, again -> https://lpalombo.cartodb.com/tables/portland_police_gang_violence_calls_2015_cdb/public
screen shot 2015-09-25 at 17 44 10

### PR DESCRIPTION
**Pull request by matallo**
close #5770

find screenshots for public map/dataset/datasets

screen shot 2015-09-28 at 14 11 45
screen shot 2015-09-28 at 14 12 07
screen shot 2015-09-28 at 14 12 46

also, consolidated behaviour on mobile (hiding username), and link to user feed in public dataset view

CR @javierarce

### CODE REVIEW

**Comment by matallo** [comment]
retest this, please

**Comment by javierarce** [comment]
:+1: :flags:

File: app/views/admin/visualizations/public_dataset.html.erb [hunk 1]
```diff
@@ -109,11 +109,11 @@
           </ul>
 
           <ul class="Navmenu-list Navmenu-list--owner">
-            <li class="Navmenu-item last-child">
-              <a href="<%= CartoDB.url(self, 'root', {}, @visualization.user) %>" class="Navmenu-link Navmenu-link--owner"><%= @name %></a>
+            <li class="Navmenu-item u-hideOnTablet last-child">
+              <a href="<%= CartoDB.url(self, 'public_user_feed_home', {}, @visualization.user) %>" class="Navmenu-link Navmenu-link--owner"><%= @name %></a>
```
Comment (xavijam) [inline app/views/admin/visualizations/public_dataset.html.erb]: I'd be great when we cut of the names/titles we add a title property with the complete string.

Comment (matallo) [inline app/views/admin/visualizations/public_dataset.html.erb]: well spotted, fixed in https://github.com/CartoDB/cartodb/pull/5808

**Pull request merged**